README.md

```diff
@@ -47,6 +49,31 @@ cd usdr-lib
 
 #### Dependencies
 
+##### Ubuntu 18.04
+
+```shell
+sudo apt install build-essential dwarves -y
+sudo apt install libusb-1.0-0-dev check dkms curl-y
```

Comment:
There is an error here, there is no space between `curl` and `-y`
Also it would be better to write as one command, as example:
```sudo apt install build-essential dwarves libusb-1.0-0-dev check dkms curl -y```

Reply:
✅